### PR TITLE
Upload server log after running Cypress tests

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

**Review by edewit — APPROVED** [review]

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ jobs:
 
       - name: Start Keycloak Server
         working-directory: ${{ env.KEYCLOAK_SERVER_BIN }}
-        run: ./kc.sh start-dev --features=admin2,admin-fine-grained-authz,declarative-user-profile &
+        run: ./kc.sh start-dev --features=admin2,admin-fine-grained-authz,declarative-user-profile &> ~/sever.log &
```
Comment (jenny-s51) [inline .github/workflows/cypress.yml:55]:
```suggestion
        run: ./kc.sh start-dev --features=admin2,admin-fine-grained-authz,declarative-user-profile &> ~/server.log &
```

File: .github/workflows/cypress.yml [hunk 2]
```diff
@@ -86,3 +86,9 @@ jobs:
         with:
           name: cypress-screenshots
           path: ${{ env.ADMIN_UI_PATH }}/assets/screenshots
+
+      - name: Upload sever logs
```
Comment (jenny-s51) [inline .github/workflows/cypress.yml:90]:
```suggestion
      - name: Upload server logs
```

File: .github/workflows/cypress.yml [hunk 3]
```diff
@@ -86,3 +86,9 @@ jobs:
...
+        uses: actions/upload-artifact@v2
+        with:
+          name: sever-log
```
Comment (jenny-s51) [inline .github/workflows/cypress.yml:93]:
```suggestion
          name: server-log
```

File: .github/workflows/cypress.yml [hunk 4]
```diff
@@ -86,3 +86,9 @@ jobs:
...
+          path: ~/sever.log
```
Comment (jenny-s51) [inline .github/workflows/cypress.yml:94]:
```suggestion
          path: ~/server.log
```

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
Looks good @jonkoops - just some quick fixes on a small typo

**Comment by jonkoops** [comment]
Thanks @jenny-s51, I've fixed the typos :+1:

**Review by jenny-s51 — APPROVED** [review]
LGTM @jonkoops ! 👍

**Pull request merged**